build/cmake/lib/webview_chromium/CMakeLists.txt

```diff
@@ -33,13 +31,23 @@ find_path(CEF_ROOT
     DOC "CEF Binary Root directory"
     )
 
-set(CEF_DOWNLOAD_DIR ${CMAKE_CURRENT_BINARY_DIR}/cef-download)
-set(CEF_SOURCE_DIR ${CMAKE_CURRENT_BINARY_DIR}/cef-source)
-set(CEF_BUILD_DIR ${CMAKE_CURRENT_BINARY_DIR}/cef-build)
+# We may need to create multiple arch-specific directories, so define the
+# variables containing the common root for all of them.
+set(CEF_DOWNLOAD_ROOT ${CMAKE_CURRENT_BINARY_DIR}/cef-download)
+set(CEF_SOURCE_ROOT ${CMAKE_CURRENT_BINARY_DIR}/cef-source)
+set(CEF_BUILD_ROOT ${CMAKE_CURRENT_BINARY_DIR}/cef-build)
 
-if(NOT CEF_ROOT)
-    message("Downloading CEF binary distribution...")
+# But when we don't need multiple directories, just use them directly.
+set(CEF_DOWNLOAD_DIR ${CEF_DOWNLOAD_ROOT})
+set(CEF_SOURCE_DIR ${CEF_SOURCE_ROOT})
+set(CEF_BUILD_DIR ${CEF_BUILD_ROOT})
```

Comment:
Variable is never used. Either it is not needed, or you use `CEF_BUILD_ROOT` incorrectly.

Reply:
Same as above.